### PR TITLE
Improve the Guest user article

### PR DESCRIPTION
**Pull request by elkinmilen**

### CODE REVIEW

File: implementer-guide/user-management/guest-user.md [hunk 1]
```diff
@@ -1,37 +1,41 @@
...
+description: Explains the Guest user feature and its purpose and limitations
 slug: guest-user
-tags: users,guest
+tags: users,guest,share,embed
 published: True
 position: 400
 ---
 
 # Guest user
+The Guest user is a built-in user account That is created upon the initial setup of the Telerik Report Server. The Guest's purpose is to be used for report document preview and sharing, either using a sharing link to a report, a report viewer control embedded in a client application, or the reports API.
```
Comment (ighristov) [inline implementer-guide/user-management/guest-user.md:12]: "... user account That is created..." => lowercase on "That"?

File: implementer-guide/user-management/guest-user.md [hunk 2]
```diff
@@ -1,37 +1,41 @@
...
 
-Guest user is a special built-in user which is created upon the initial setup of Telerik Report Server. The Guest's purpose is to be used for read-only connections to report server, either using the reports API or sharing a link to a report.
+## Guest User Administration
+The Guest user is not enabled by default. To enable it, navigate to the _User Management -> Users_ view, select the Guest user from the list of users and turn on the _Enabled_ switch.
```
Comment (ighristov) [inline implementer-guide/user-management/guest-user.md:15]: reading about the Enabled switch leaves a sense of mismatching UI - I would expect to see something like a switch (toggle) control, but I see a simple checkbox instead.
"... check the Enabled checkbox" or "tick the Enabled checkbox (to avoid repetition)" would do fine, IMHO.

File: implementer-guide/user-management/guest-user.md [hunk 3]
```diff
@@ -1,37 +1,41 @@
...
+###	Share a report that resides on the Report Server with a URL
+Report Server's API provides a method that serves a report by specified category, report name, and report revision. The format of the method is 
+*{serverUrl}/report/share/{category}/{reportName}/{revision}*. The _revision_ parameter can be omitted, 
+which will result in displaying the last revision of the specified report. In case the Guest account is disabled, a 403-Forbidden error is returned.
 
--	Cannot be edited - since it doesn't have any user-related fields except *username*, the **Edit** button is disabled when the Guest user is selected. Its Enabled/Disabled status can be changed using a button in the Permissions tab.
+###	Display a server report in an embedded Report Viewer control
+The Guest account can be used for authentication against the Report Server when you want to embed any Report Viewer displaying server reports into a user application.
+To do this no authentication token is provided and Guest user authorization is done on the server. 
+You can read more in the [Integration With Report Viewers](%slug integration-with-report-viewers%) article.
```
Comment (ighristov) [inline implementer-guide/user-management/guest-user.md:28]: I think the %slug% should be embraced in curly braces.

File: implementer-guide/user-management/guest-user.md [hunk 4]
```diff
@@ -1,37 +1,41 @@
...
+## Usage
+###	Share a report that resides on the Report Server with a URL
+Report Server's API provides a method that serves a report by specified category, report name, and report revision. The format of the method is 
+*{serverUrl}/report/share/{category}/{reportName}/{revision}*. The _revision_ parameter can be omitted, 
+which will result in displaying the last revision of the specified report. In case the Guest account is disabled, a 403-Forbidden error is returned.
 
--	Cannot be edited - since it doesn't have any user-related fields except *username*, the **Edit** button is disabled when the Guest user is selected. Its Enabled/Disabled status can be changed using a button in the Permissions tab.
+###	Display a server report in an embedded Report Viewer control
+The Guest account can be used for authentication against the Report Server when you want to embed any Report Viewer displaying server reports into a user application.
+To do this no authentication token is provided and Guest user authorization is done on the server.
```
Comment (DimitarNikolovv06) [inline implementer-guide/user-management/guest-user.md:27]: maybe try wording it as:

This doesn't require sending an authentication token from the report viewer as the user authorization is performed by the server.

File: implementer-guide/user-management/guest-user.md [hunk 5]
```diff
@@ -1,37 +1,41 @@
...
 position: 400
 ---
 
 # Guest user
+The Guest user is a built-in user account That is created upon the initial setup of the Telerik Report Server. The Guest's purpose is to be used for report document preview and sharing, either using a sharing link to a report, a report viewer control embedded in a client application, or the reports API.
 
-Guest user is a special built-in user which is created upon the initial setup of Telerik Report Server. The Guest's purpose is to be used for read-only connections to report server, either using the reports API or sharing a link to a report.
+## Guest User Administration
+The Guest user is not enabled by default. To enable it, navigate to the _User Management -> Users_ view, select the Guest user from the list of users and turn on the _Enabled_ switch.
+By default, this user has _Read_ permission on all reports in all categories. If some reports or categories should not be generally accessible, delete the default permission and substitute it with more specific _Read_ permissions.
```
Comment (DimitarNikolovv06) [inline implementer-guide/user-management/guest-user.md:16]: consider emphasizing the Read work with instead ` ` , example of how it will look:  `Read`

File: implementer-guide/user-management/guest-user.md (hunk 3 above)
Comment (DimitarNikolovv06) [inline implementer-guide/user-management/guest-user.md:28]: generally, it is advised to not use "you" when possible, may try like this:

"More information on this topic is available in the"

**Review by ighristov — APPROVED** [review]
Thanks for improving this article!
Left a few minor comments.

File: implementer-guide/user-management/guest-user.md (hunk 1 above)
Comment (elkinmilen) [reply implementer-guide/user-management/guest-user.md:12]: Shame. Fixed

File: implementer-guide/user-management/guest-user.md (hunk 2 above)
Comment (elkinmilen) [reply implementer-guide/user-management/guest-user.md:15]: Better. Applied. Thanks!

File: implementer-guide/user-management/guest-user.md (hunk 3 above)
Comment (elkinmilen) [reply implementer-guide/user-management/guest-user.md:28]: Ahaa. Right. Thanks!

File: implementer-guide/user-management/guest-user.md (hunk 4 above)
Comment (elkinmilen) [reply implementer-guide/user-management/guest-user.md:27]: Well. the second is not a consequence of the first. The idea is that if you do not send an auth. token the request is authenticated as the Guest user. Then, the server applies the authorization based on the permissions given to the user, Guest in this case. I will rephrase it to make it more clear. Thanks.

File: implementer-guide/user-management/guest-user.md (hunk 5 above)
Comment (elkinmilen) [reply implementer-guide/user-management/guest-user.md:16]: Better. Applied. Thanks

File: implementer-guide/user-management/guest-user.md (hunk 3 above)
Comment (elkinmilen) [reply implementer-guide/user-management/guest-user.md:28]: Fixed

File: implementer-guide/user-management/guest-user.md [hunk 6]
```diff
@@ -1,37 +1,41 @@
...
 
--	Cannot be edited - since it doesn't have any user-related fields except *username*, the **Edit** button is disabled when the Guest user is selected. Its Enabled/Disabled status can be changed using a button in the Permissions tab.
+###	Display a server report in an embedded Report Viewer control
+The Guest account can be used for authentication against the Report Server when you want to embed any Report Viewer displaying server reports into a user application.
+To do this no authentication token is provided and Guest user authorization is done on the server. 
+You can read more in the [Integration With Report Viewers](%slug integration-with-report-viewers%) article.
 
--	Cannot be used to log in - the Guest user doesn't have a password, so it cannot be used to log in into Telerik Report Server.
+## Limitations of the Guest user
+-	Cannot be edited - since it doesn't have any user-related fields except *username*, the _Edit_ button is disabled when the Guest user is selected.
```
[the remark this answers is not in the document]
Comment (elkinmilen) [reply implementer-guide/user-management/guest-user.md:31]: Done, thanks!
